primitives/src/identity_id.rs

```diff
@@ -3,6 +3,8 @@ use core::fmt::{Display, Formatter};
 use core::str;
 use rstd::prelude::*;
 use runtime_primitives::traits::Printable;
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
```

Comment:
For consistency, let's use `sr_primitives::{Deserialize, Serialize};`

Reply:
Agreed. I've replaced `serde` serializers with `sr-primitive`'s.